src/Explorer/SplashScreen/SplashScreen.tsx

```diff
@@ -1,63 +1,161 @@
+import * as Constants from "../../Common/Constants";
+import { Link } from "office-ui-fabric-react/lib/Link";
 import NewContainerIcon from "../../../images/Hero-new-container.svg";
 import NewNotebookIcon from "../../../images/Hero-new-notebook.svg";
 import NewQueryIcon from "../../../images/AddSqlQuery_16x16.svg";
 import OpenQueryIcon from "../../../images/BrowseQuery.svg";
 import NewStoredProcedureIcon from "../../../images/AddStoredProcedure.svg";
 import ScaleAndSettingsIcon from "../../../images/Scale_15x15.svg";
-import { SplashScreenComponent, SplashScreenItem } from "./SplashScreenComponent";
 import * as MostRecentActivity from "../MostRecentActivity/MostRecentActivity";
 import AddDatabaseIcon from "../../../images/AddDatabase.svg";
 import SampleIcon from "../../../images/Hero-sample.svg";
 import { DataSamplesUtil } from "../DataSamples/DataSamplesUtil";
 import Explorer from "../Explorer";
 import { userContext } from "../../UserContext";
+import { FeaturePanelLauncher } from "../Controls/FeaturePanel/FeaturePanelLauncher";
 
-/**
- * TODO Remove this when fully ported to ReactJS
- */
-export class SplashScreenComponentAdapter implements ReactAdapter {
+export interface SplashScreenItem {
+  iconSrc: string;
+  title: string;
+  info?: string;
+  description: string;
+  onClick: () => void;
+}
+
+export interface SplashScreenProps {
+  explorer: Explorer;
+}
+
+export class SplashScreen extends React.Component<SplashScreenProps> {
+  private static readonly seeMoreItemTitle: string = "See more Cosmos DB documentation";
+  private static readonly seeMoreItemUrl: string = "https://aka.ms/cosmosdbdocument";
   private static readonly dataModelingUrl = "https://docs.microsoft.com/azure/cosmos-db/modeling-data";
   private static readonly throughputEstimatorUrl = "https://cosmos.azure.com/capacitycalculator";
   private static readonly failoverUrl = "https://docs.microsoft.com/azure/cosmos-db/high-availability";
 
-  public parameters: ko.Observable<number>;
+  private readonly container: Explorer;
 
-  constructor(private container: Explorer) {
-    this.parameters = ko.observable<number>(Date.now());
-    this.container.tabsManager.openedTabs.subscribe((tabs) => {
-      if (tabs.length === 0) {
-        this.forceRender();
-      }
-    });
-    this.container.selectedNode.subscribe(this.forceRender);
-    this.container.isNotebookEnabled.subscribe(this.forceRender);
+  constructor(props: SplashScreenProps) {
+    super(props);
+    this.container = props.explorer;
+    this.container.tabsManager.openedTabs.subscribe(() => this.setState({}));
+    this.container.selectedNode.subscribe(() => this.setState({}));
+    this.container.isNotebookEnabled.subscribe(() => this.setState({}));
   }
 
-  public forceRender = (): void => {
-    window.requestAnimationFrame(() => this.parameters(Date.now()));
-  };
+  public shouldComponentUpdate() {
+    return this.container.tabsManager.openedTabs.length === 0;
```

Comment:
Note that there's some talk to move splash screen to a tab. But it's easy to remove later.